src/testdrive/src/action.rs

```diff
@@ -553,7 +556,10 @@ fn substitute_vars(msg: &str, vars: &HashMap<String, String>) -> Result<String,
     let mut err = None;
     let out = RE.replace_all(msg, |caps: &Captures| {
         let name = &caps[1];
-        if let Some(val) = vars.get(name) {
+        if name == INGEST_ITERATION_RUNTIME_VAR {
```

Comment:
To make this less special casey, can we make `substitute_vars` take in a prefix of variables to ignore? And then each command can pass in its own name. So, e.g., `kafka-ingest` would ignore substituting any variables that started with `kafka-ingest.`.